examples/hb-subset.example.node.js

```diff
@@ -20,15 +51,38 @@ const SUBSET_TEXT = 'abc';
     const blob = exports.hb_blob_create(fontBuffer, fontBlob.byteLength, 2/*HB_MEMORY_MODE_WRITABLE*/, 0, 0);
     const face = exports.hb_face_create(blob, 0);
     exports.hb_blob_destroy(blob);
+    // TODO: get gids via hb-shape
+    const SUBSET_GIDS = [4261]; // star icon
+    const SUBSET_TEXT = ['star']
 
     /* Add your glyph indices here and subset */
     const input = exports.hb_subset_input_create_or_fail();
+    const glyph_set = exports.hb_subset_input_glyph_set(input);
+    for (const gid of SUBSET_GIDS) {
+        exports.hb_set_add(glyph_set, gid);
+    }
+
+    const gids = closureAndGetGids(input);
+    console.log('📌 Step 1: Glyph IDs', `[${gids.join(', ')}]`)
+
+    const glyph_set2 = exports.hb_subset_input_glyph_set(input);
+    for (const gid of gids) {
+        exports.hb_set_add(glyph_set2, gid);
+    }
+
     const unicode_set = exports.hb_subset_input_unicode_set(input);
-    for (const text of SUBSET_TEXT) {
+    for (const text of SUBSET_TEXT.toString()) {
         exports.hb_set_add(unicode_set, text.codePointAt(0));
```

Comment:
Should this add all codepoints in text? Currently looks like it only adds the first.

Reply:
Here it loops through each character, and I've printed out the codePoints:

// SUBSET_TEXT = ['star']

```
codePoints [ 115, 116, 97, 114 ]
```